wzdx/raw_to_standard/planned_events.py

```diff
@@ -311,15 +318,15 @@ def create_rtdh_standard_msg(pd):
                 "end_timestamp": date_tools.date_to_unix(end_date),
             },
             "detail": {
-                "road_name": pd.get("properties/routeName"),
-                "road_number": pd.get("properties/routeName"),
+                "road_name": roadName,
+                "road_number": roadName,
```

Comment:
Is this "road_number" expected to be numeric, and if so will we ever run into issues if we get a road name that isn't?

Reply:
road_number ideally should be numeric, like I-70 instead of interstate 70 or something like that. I have not added this logic yet, just have the road_number field for improvement in the future. The later code only adds the road_number if it is different from the road_name